cmdstanpy/model.py

```diff
@@ -941,19 +941,22 @@ def sample(
             parallel_procs = parallel_chains
             num_threads = threads_per_chain
             one_process_per_chain = True
+            assert isinstance(self.exe_file, str)  # make typechecker happy
+            info_dict = self.exe_info()
+            stan_threads = info_dict.get('STAN_THREADS')
+            if stan_threads is not None:
```

Comment:
I prefer using the default value functionality of `get` rather than the extra if case. `stan_threads = info_dict.get('STAN_THREADS', 'false').lower()`